src/main/scala/de/htwg/se/sudoku/aview/gui/CellPanel.scala

```diff
@@ -70,7 +85,8 @@ class CellPanel(row: Int, column: Int, controller: ControllerInterface) extends
 
   def redraw = {
     contents.clear()
-    if ((controller.isShowCandidates(row, column) || controller.showAllCandidates) && !controller.isSet(row, column)) {
+    if ((controller.isShowCandidates(row, column) || controller.showAllCandidates) && !controller
```

Comment:
this also looks strange, it would be cleaner to begin a new line with every logical operator (|| / &&). meh @scalafmt

Reply:
Same as above...